package.json

```diff
@@ -8,8 +8,7 @@
 	"scripts": {
 		"lint": "eslint .",
 		"build": "rollup -c",
-		"prepublishOnly": "npm run build",
-		"test": "jest"
```

Comment:
What do you think about removing the test files in another commit? I think it'd easy to revert it in the future.

Reply:
Done :smile: